BlazarUI/app/scripts/data/StoredBuilds.js

```diff
@@ -1,12 +1,12 @@
 //
 // Super class for polling the builds store
-//  - extended by BranchesApi abnd BranchBuildsApi
+//  - extended by BranchBuildsApi
```

Comment:
technically HostsApi extends this too but I don't think it's necessary to mention extended by ... here